library-best-practices/JLBP-6.md

```diff
@@ -128,24 +132,26 @@ is used. Let's take examples from two extremes.
    function with one reference in a leaf of the dependency graph that is deleted
    between major version 1 to major version 2.
 
-In this case, moving 10,000 references to a new package in a large dependency
-tree would be a very expensive endeavor. In contrast, updating the one place
-where the old function is used to use the new function instead would be
-considerably less work and could be rolled out much more quickly. In this
-scenario, it is clearly superior to keep the same Java package.
+   In this case, moving 10,000 references to a new package in a large dependency
+   tree would be a very expensive endeavor. In contrast, updating the one place
+   where the old function is used to use the new function instead would be
+   considerably less work and could be rolled out much more quickly. In this
+   scenario, it is clearly superior to keep the same Java package.
 
 2. A library with 10,000 references throughout 100 packages, and a large
    refactoring breaks the surface of 5,000 of those references between major
    version 1 to major version 2.
```

Comment:
The comparison of the two examples is not clear to me.

Example 1
> a function with one reference in a leaf of the dependency graph that is deleted between major version 1 to major version 2.

Example 2
>  a large refactoring breaks the surface of 5,000 of those references between major version 1 to major version 2.


Probably because some word only appear in one of them: "leaf", "dependency graph that is deleted", "a large refactoring", "breaks the surface"

I'm thinking of a way to make their comparison clearer using the same set of words among them.